### DISCUSSION TITLE
a simple outpaint/inpaint tab

### DISCUSSION THREAD

**Discussion by yesbroc** [Stability-AI/StableSwarmUI#97] [Ideas]
like the dall-e 2 outpaint grid, where you can input more than one image and sort of merge/bridge them together or just general editing and outpainting

**Comment by mcmonkey4eva** [comment]
Very much a planned feature, taking some careful and thorough planning first, we want to do it right.

**Reply by yesbroc** [reply]
could it be possible to integrate a sketch function in the grid? to help guide the creation of images, inpainting/outpainting with color
